app/models/facility.rb

```diff
@@ -124,20 +124,23 @@ def organization_exists
 
   def facility_group_exists
     organization = Organization.find_by(name: organization_name)
-    facility_group = FacilityGroup.find_by(name: facility_group_name,
-                                           organization_id: organization.id) if organization.present?
```

Comment:
👍 I rarely use the suffix `if / unless`, and I know it can be hard to catch sometimes as line length creeps up.